acrobat/scripts/scripts.js

```diff
@@ -111,6 +111,12 @@ if (hostname === 'www.stage.adobe.com') {
   document.head.appendChild(tokenElement);
 }
 
+// Adding .html to canonical url for .ing pages
+if (hostname.endsWith('.ing')) {
+  const canonEl = document.head.querySelector('link[rel="canonical"]');
+  canonEl?.setAttribute('href', `${canonEl.href}.html`);
```

Comment:
What if there is `.html` already?
I checked https://edit.ing/acrobat/online/sign-pdf and it seems there is already `.html`
<img width="947" alt="image" src="https://github.com/adobecom/dc/assets/55804872/041cd4a2-9f0d-4a8e-afca-5c439bc8276d">

Reply:
Yeah, you're right, just in case I need to check if it ends with `.html`. On that page there is `.html` because my previous solution works on that link.